src/App.tsx

```diff
@@ -340,6 +340,20 @@ export default function App() {
     }
   }, [type]);
 
+  const [locale] = useState('en');
```

Comment:
Default the locale to 'en' for now. This will be changed later when we get to the languages portion of i18n.

Reply:
Default sounds good 👍 
Thoughts on moving the timezones fetching etc useEffect into the LocalizationProvider - separate the responsibility?
The user and org stuff we left them in the App because it was already tightly coupled with the rest of the code.